toffy/watcher_callbacks.py

```diff
@@ -148,9 +166,33 @@ def generate_qc(self, qc_out_dir: str, panel: pd.DataFrame = None, **kwargs):
         for metric_name, data in metric_data.items():
             data.to_csv(os.path.join(qc_out_dir, metric_name), index=False)
 
+    def generate_mph(self, mph_output_dir, panel: pd.DataFrame = None):
```

Comment:
the output folder argument names in `plot_mph_metrics` and `generate_mph` need to match, i.e `mph_output_dir` should be `mph_out_dir` or vice-versa